core/src/test/java/org/jdbi/v3/core/EnumsConfigTest.java

```diff
@@ -70,6 +71,39 @@ public void namesAreMappedCorrectly() {
         });
     }
 
+        @Test
+        public void customizedNamesAreBoundCorrectly() throws NoSuchFieldException {
+        db.getJdbi().useHandle(h -> {
+            h.createUpdate("create table enums(id int, name varchar)").execute();
+
+            h.createUpdate("insert into enums (id, name) values (1, :name)")
+                    .bind("name", Foobar.CUSTOM)
+                    .execute();
+
+            String ordinal = h.createQuery("select name from enums")
+                    .mapTo(String.class)
+                    .one();
+
+            assertThat(ordinal)
+                    .isEqualTo(Foobar.class.getField(Foobar.CUSTOM.name())
+                            .getAnnotation(DatabaseValue.class).value());
```

Comment:
I think it's better for a test to be explicit about expected values rather than using calculated values where possible. This test reads more like `assert impl() == impl()` than it should.

Reply:
Right. I did it like this because the two methods the new tests are based on looked quite dynamic as well.

Reply:
I don't really follow. Do you object to changing it to `isEqualTo("CUST")`?